experimental/packages/otlp-grpc-exporter-base/src/grpc-exporter-transport.ts

```diff
@@ -150,7 +155,9 @@ export class GrpcExporterTransport implements IExporterTransport {
         });
       }
 
-      this._client.export(
+      // Using `any` as the gRPC client constructor is created on runtime,
+      // so we don't have any types for the resulting client.
+      (this._client as any).export(
```

Comment:
I think there is a directive to make tslint ignore this warning

Reply:
Added in [971d215](https://github.com/open-telemetry/opentelemetry-js/pull/4612/commits/971d21565fd0c7f9487ba3dd8b6b8d8c4b614831)